internal/app/referenceserver/impl.go

```diff
@@ -55,8 +55,8 @@ func (s *conformanceServer) Unary(
 		[]*anypb.Any{msgAsAny},
 	)
 	if connectErr != nil {
-		internal.AddHeaders(req.Msg.ResponseDefinition.ResponseHeaders, connectErr.Meta())
-		internal.AddHeaders(req.Msg.ResponseDefinition.ResponseTrailers, connectErr.Meta())
+		internal.AddHeaders(req.Msg.ResponseDefinition.GetResponseHeaders(), connectErr.Meta())
+		internal.AddHeaders(req.Msg.ResponseDefinition.GetResponseTrailers(), connectErr.Meta())
```

Comment:
This was "just in case", since I had played around with `buf curl` and an empty request to troubleshoot TLS issues. If the error from `parseUnaryResponseDefinition` is that it is complaining that no response definition is provided, these would cause the server to panic and fail to actually report that error.